test/fun_prop_dev.lean

```diff
@@ -42,7 +48,7 @@ set_option linter.unusedVariables false
 
 -- Lin is missing `const` theorem
 @[fun_prop] theorem Lin_id : Lin (fun x : α => x) := silentSorry
-@[fun_prop] theorem Lin_const {β} [Obj β] [Zero β] : Lin (fun x : α => (0 : β)) := silentSorry
+@[fun_prop] theorem Lin_const {β} [Zero β] : Lin (fun x : α => (0 : β)) := silentSorry
```

Comment:
Why do you need to remove `Obj` here?

Reply:
that's because it is not defined in this branch. it used to be there in master, but Thomas Skrivan who works on the tactic removed it for the port.

Reply:
All fixed.

Reply:
it *was* not defined on this branch